src/main/java/wolox/bootstrap/models/User.java

```diff
@@ -1,22 +1,11 @@
 package wolox.bootstrap.models;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.postgresql.shaded.com.ongres.scram.common.util.Preconditions;
+
+import javax.persistence.*;
```

Comment:
Same here

Reply:
Done but in the next PR: https://github.com/Wolox/java-bootstrap/pull/30